apps/io-wallet-user-func/src/app/main.ts

```diff
@@ -117,6 +117,7 @@ app.http("createWalletInstance", {
   handler: withAppInsights(
     CreateWalletInstanceFunction({
       attestationService: mobileAttestationService,
+      mail: config.mail,
```

Comment:
?

Reply:
The email must not be a parameter of this function. You can write a method that returns the email from the CF by mocking it for now

Reply:
you could create an abstract service to send emails.

Reply:
What do you mean by "abstract service"? Is it a part of a hexagonal architecture?

Reply:
I simply wouldn't send the mail configuration but a service that allows you to send emails that you can use regardless of the email provider